Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -687,6 +688,10 @@ def send_request(self, path: str, method: str = 'GET', body: dict | None = None,
                 'Accept': 'application/json',
                 'Content-Type': 'application/json'
             }
+        # The attachments table does not support v2 api version
+        if get_attachments:
```

Comment:
Does this have any impact on customers who are expecting to use v2 in any way?

Reply:
No, it shouldn't.